updated documentation for event groups (#2958)
nasa-gcn/gcn.nasa.gov: app/routes/docs.circulars.archive.mdx

You can search the Circulars Archive by keywords or phrases in the subject, body, and submitter fields. By default, results will be returned in reverse chronological order, but you can also choose to sort the results by relevance. You can navigate through the search results using pagination controls and specify the number of results to display per page. By default, 100 results are displayed per page.

You can filter search results by submission date. Select the dropdown button labeled "Filter by date". From there, you can either select from a number of predefined time ranges (e.g., "Last Hour", "Last Day", etc.) or specify a custom date range, which can either be set as a start date, an end date, or both. All search filters can be cleared by clicking the 'x' button in the search bar.

> You can enter a specific Circular ID (e.g. 12345) and it will list that Circular. Note that GRB names without a letter can be interpreted as Circular IDs.

Reviewer: This doesn't depend on synonyms, does it? Would you please pull this out as a separate PR?

<WithFeature SYNONYMS>
## Event View

The Circulars archive provides an Event View that allows users to view all Circulars associated with a particular astronomical event.

Each Circular is automatically assigned an event name derived from the subject (e.g. GRB 250109A, EP250109a). This assignment is done by matching against a regular expressions list.

> The event name is listed in the JSON format version of Circulars. The first event name used in the subject is associated automatically, and all Circulars with the same event name are grouped. GCN moderators can associate multiple event names to groups, such that all Circulars with these event names will be listed in the same event group.

Reviewer: ```suggestion
The event name is listed in the JSON format version of Circulars. The first event name used in the subject is associated automatically, and all Circulars with the same event name are grouped. GCN moderators can associate multiple event names with a group, such that all Circulars with these event names will be listed in the same event group.
```

### Adding events to the same group

> GCN Team moderators can create or modify groups. If you would like to suggest the creation or modification of an event group, submit a ticket via [the contact form](/support).

Reviewer: ```suggestion
GCN Team moderators can create or modify event groups. If you would like to suggest the creation or modification of an event group, submit a ticket via [the contact form](/support).
```
